tests/jerry/number-prototype-to-fixed.js

```diff
@@ -31,14 +31,16 @@ assert((-Infinity).toFixed(0) === "-Infinity");
 assert(NaN.toFixed(0) === "NaN");
 assert((0.0).toFixed(0) === "0");
 assert((0.0).toFixed(1) === "0.0");
-assert((-0.0).toFixed(0) === "-0");
```

Comment:
According to **http://www.ecma-international.org/ecma-262/5.1/#sec-15.7.4.5**

```
If x < 0, then
Let s be "-".
Let x = –x.
```
And `-0.0 < 0.0` gives false because they are equal.